reconcile/test/test_status_board.py

```diff
@@ -19,6 +19,7 @@
 
 
 class TestStatusBoard(AbstractStatusBoard):
+    __test__ = False
```

Comment:
Remove this please, we'll handle it in a different PR

Reply:
dito